modules/brew.bash

```diff
@@ -15,12 +15,18 @@ info_echo "Install Brew formalue"
 brew tap "Homebrew/bundle" 2> /dev/null
 brew bundle --file="$osx_bootstrap/Brewfile"
 
+# Linking Qt to have 'qmake' for building Capybara
+brew link --force qt@5.5
+
 # https://github.com/eventmachine/eventmachine/issues/602#issuecomment-152184551
 info_echo "Link keg-only openssl to /usr/local to let software outside of Homebrew to find it"
 brew unlink openssl && brew link openssl --force
 
 info_echo "Link Curl with openssl"
 brew link --force curl
 
+info_echo "Link readline to correctly build Ruby later"
+brew link --force readline
```

Comment:
Could we try to link these formulae in the [`Brewfile`](https://github.com/Homebrew/homebrew-bundle#usage)